README.md

```diff
@@ -55,7 +55,7 @@ import com.dylibso.chicory.runtime.Instance;
 import java.io.File;
 
 // point this to your path on disk
-Module module = Module.build(new File("./factorial.wasm"));
+Module module = Module.builder(new File("./factorial.wasm")).build();
```

Comment:
Maybe we want to recommend this pattern?:

```suggestion
Module module = Module.builder("./factorial.wasm").build();
```

Reply:
I think we should keep the `File` here, as `factorial.wasm` needs to be available on the class path to be able to load it with your suggestion.